lib/pages/onboarding_screen.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+
+import 'package:introduction_screen/introduction_screen.dart';
+import 'package:safety/pages/login_page.dart';
+
+class OnboardingScreen extends StatelessWidget {
+  final List<PageViewModel> listPageViewModels = [
+    PageViewModel(
```

Comment:
Let me provide you with screenshots that will go here.

Reply:
sure